login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 계정</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="common.css" />
+    <link rel="stylesheet" href="login.css" />
+  </head>
+  <body>
+    <main class="main">
+      <a href="/" class="main__logo">
+        <img src="img/logo.svg" alt="판다마켓 로고">
+        판다마켓
+      </a>
+      <article class="main__login-wrap">
+        <form class="login__form">
+          <div class="form__item">
+            <label class="form__label" for="login--id">이메일</label>
+            <input
+              class="form__input"
+              placeholder="이메일을 입력해주세요"
+              type="text"
+              id="login--id"
+              name="loginId"
```

Comment:
```suggestion
              id="email"
              name="email"
```

name필드의 경우 폼 제출시 제출된 데이터의 키값으로 활용되어요.
따라서 바로 직관적으로 볼 수 있는 이름을 적용해주는게 좋답니다